kinder/pkg/cluster/manager/actions/kubeadm-join.go

```diff
@@ -139,9 +140,10 @@ func kubeadmJoinControlPlaneWithPhases(cp *status.Node, patchesDir, ignorePrefli
 		fmt.Sprintf("--config=%s", constants.KubeadmConfigPath),
 		fmt.Sprintf("--v=%d", vLevel),
 	}
-
```

Comment:
in general, we should avoid whitespace changes like adding lines and removing lines as part of other code changes.